hvpy/facade.py

```diff
@@ -382,3 +384,57 @@ def queueMovie(
         reqObservationDate=reqObservationDate,
     )
     return execute_api_call(input_parameters=params)
+
+
+@add_shared_docstring(reQueueMovieInputParameters)
+def reQueueMovie(
+    id: str,
+    force: Optional[bool] = False,
+) -> Union[bytes, str, Dict[str, Any]]:
+    """
+    Re-generate a custom movie that is no longer cached on the server.
+
+    Parameters
+    ----------
+    {Insert}
+    Examples
+    --------
+    >>> from hvpy import reQueueMovie
+    >>> reQueueMovie(id="gxRN5", force=True)
+    {'id': 'gxRN5', 'eta': 274, 'queue': 0, 'token': '...'}
+    """
+    params = reQueueMovieInputParameters(
+        id=id,
+        force=force,
+    )
+    return execute_api_call(input_parameters=params)
+
+
+@add_shared_docstring(getMovieStatusInputParameters)
+def getMovieStatus(
+    id: str,
+    format: str,
+    verbose: Optional[bool] = False,
+    callback: Optional[str] = None,
+    token: Optional[str] = None,
+) -> Union[bytes, str, Dict[str, Any]]:
+    """
+    Get the status of a movie.
+
+    Parameters
+    ----------
+    {Insert}
+    Examples
+    --------
+    >>> from hvpy import getMovieStatus
+    >>> getMovieStatus(id="VXvX5", format="mp4")
```

Comment:
Will this ID always be there?

Reply:
Movies that are more than 90 days old are deleted, but they can be re-built. The ID should always be there as a reference to movie.